transcom/mymove
# B 18546 Office user not alter order type for safety move

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A897668&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary
The office user should not be able to alter the Order type after it has already been created as a Safety move.  Also, if a move has been created as a non Safety move, it cannot change to a Safety move.

### Setup to Run the Code

You will need to work with the FEATURE_FLAG_SAFETY_MOVE on.
You will need to have a Service Counselor, TOO and TIO user with safety privileges. 

### How to test

1. Login as a Service Counselor and create or choose an existing Safety move.
2. Once on the Move details page, click on the View and edit orders button.
3. Verify that in the Orders type dropdown, you see Safety selected and the dropdown is disabled and not allow the user to change it.
4. Find a move that is not a Safety move, navigate to the View orders page and verify that you do not see Safety as an option in the Orders type dropdown.
5. Login as a TOO, run through steps 1-4.
6. Login as a TIO, run through steps 1-4.
